core/src/main/java/be/isach/ultracosmetics/cosmetics/particleeffects/ParticleEffectSuperVillain.java

```diff
@@ -0,0 +1,120 @@
+package be.isach.ultracosmetics.cosmetics.particleeffects;
+
+import be.isach.ultracosmetics.UltraCosmetics;
+import be.isach.ultracosmetics.cosmetics.type.ParticleEffectType;
+import be.isach.ultracosmetics.player.UltraPlayer;
+import com.cryptomorin.xseries.particles.ParticleDisplay;
+import com.cryptomorin.xseries.particles.XParticle;
+import org.bukkit.Location;
+import org.bukkit.util.Vector;
+
+import java.awt.*;
+
+/**
+ * Represents an instance of super villain particles summoned by a player.
+ *
+ * @author iSach
+ * @since 11-11-2015
+ */
+public class ParticleEffectSuperVillain extends ParticleEffect {
+
+    private final boolean x = true;
+    private final ParticleDisplay secondary;
+
+    public ParticleEffectSuperVillain(UltraPlayer owner, ParticleEffectType type, UltraCosmetics ultraCosmetics) {
+        super(owner, type, ultraCosmetics);
+        display.withColor(Color.BLACK).withCount(getModifiedAmount(3));
+        secondary = ParticleDisplay.of(XParticle.ELECTRIC_SPARK).offset(0.15, 0.1, 0.15)
+                .withEntity(getPlayer()).withCount(getModifiedAmount(4));
+    }
+
+    private final boolean[][] shape = {
+            {false, x, x, x, false,},
+            {false, x, x, x, false,},
+            {x, x, x, x, x,},
+            {x, x, x, x, x,},
+            {x, x, x, x, x,},
+            {x, x, x, x, x,},
+            {x, x, false, x, x,},
+            {x, false, false, false, x,},
```

Comment:
It'd be good to stick with either variables here (`x` and something else), or `true` and `false`, and align the columns like a table so it's easier to work on